lakey_service/downloader/executors/athena.py

```diff
@@ -7,6 +7,7 @@
 import requests
 import pandas as pd
 
+from .base import Base
```

Comment:
sorry that was not clear from my comment, for consistency we should have 
```python
from .base import BaseExecutor
```
please have in mind that the absolute path of this class is `<something>.executors.base` therefore we don't need `executor` prefix / suffix in the module name but class itself is something else since its name appears in pontetially many places not only as the module name in the imports block. Therefore class name must have clear and descriptive name